helion/autotuner/logger.py

```diff
@@ -134,7 +135,10 @@ def classify_triton_exception(err: BaseException) -> Literal["raise", "warn", "d
     if isinstance(err, PTXASError) or err.__class__.__name__ == "PTXASError":
         return "warn"
 
-    msg = str(err)
+    def msg_with_cause(err: BaseException | None) -> str:
+        return str(err) + msg_with_cause(err.__cause__) if err is not None else ""
+
+    msg = msg_with_cause(err)
```

Comment:
@v0i0 would be awesome to break out changes in this file into a standalone PR to be landed first, to fix the current benchmark CI. thanks!

Reply:
took it out based on our discussion again